README.md

```diff
@@ -1,61 +1,63 @@
 # etcd.io
 
-This repository houses all of the assets used to build the etcd docs and website available at https://etcd.io.
+The [etcd.io][] website, built using [Hugo][] and hosted on [Netlify][].
 
-## Run the site locally
+## Build prerequisites
 
-### Prerequisites
+To build and serve the site, you'll need the latest [LTS release][] of **Node**.
+Like Netlify, we use **[nvm][]**, the Node Version Manager, to install and
+manage Node versions:
 
-To build and serve the site, you'll need these tools:
+```console
+$ nvm install --lts
+$ nvm use --lts
```

Comment:
Does `nvm install --lts` always result in the latest node being used? (I've tested this on two systems, and `nvm install --lts` always uses the latest, making `nvm use --lts` look a bit redundant)

Reply:
I vaguely recall that there were circumstances under which the `use` was needed, but according to https://github.com/nvm-sh/nvm/issues/2057, this is no longer the case since 2014. I'll remove the use of `use`. Nice catch!

Reply:
Done

Reply:
Ah, now a remember: I was wanting to cater to the scenario that a developer already had Node.js installed -- and so might skip the install command. But, since 2014, I see that it's ok to execute `install` whether or not the latest LTS is already installed. In either case, it is equivalent to following that command with `nvm use`.